src/test/WormholeJoin.t.sol

```diff
@@ -583,4 +583,88 @@ contract WormholeJoinTest is DSTest {
         assertEq(_ink(), 250_000 ether);
         assertEq(_art(), 250_000 ether);
     }
+
+    function testRegisterAndWithdrawPayingOperatorFee() public {
+        WormholeGUID memory guid = WormholeGUID({
+            sourceDomain: "l2network",
+            targetDomain: "ethereum",
+            receiver: address(123),
+            operator: address(this),
+            amount: 250_000 ether,
+            nonce: 5,
+            timestamp: uint48(block.timestamp)
+        });
+        assertEq(vat.dai(address(this)), 0);
+        join.requestMint(guid, 0, 10 ether / 10000);   // 0.1% fee
```

Comment:
nit: `10 ether / 10000` isn't it easier to just write `1 ether / 1000`? 🤔

Reply:
I like keeping it in bps.

Reply:
![image](https://user-images.githubusercontent.com/1814312/149170517-921a6c4a-df96-4ae7-888c-31911c504f6f.png)

Reply:
exactly